scylladb/sphinx-scylladb-theme
# Add license notice space

Related issue https://github.com/scylladb/java-driver/issues/165

Add a new section for LICENSE information after content.

## How to test this PR

1. Edit `sphinx_scylladb_theme/notice.html` and add the following content:

```
<div class="notice">
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut tincidunt sapien vitae elementum cursus.
Maecenas dignissim libero sapien, pretium posuere sem imperdiet eget.
</div>
```

2. Build the docs with `make preview`. You should see the notice after the prev / next navigation buttons on every page.

Preview:

![image](https://user-images.githubusercontent.com/9107969/200593266-b12105fa-0299-47ac-9258-fa3c70643bb1.png)


Once we merge this pull request, we can release a new patch to release a new version of the theme.

Then, we will override the `notice.html` template in the repos java-driver and python-driver.
